{"name": "test_known_bug", "status": "skipped", "statusDetails": {"message": "XFAIL Known bug in Chrome headless mode\n\nAssertionError: assert 1 == 2", "trace": "@pytest.mark.xfail(reason=\"Known bug in Chrome headless mode\")\n    def test_known_bug():\n>       assert 1 == 2\nE       assert 1 == 2\n\ntests\\test_skip_xfail.py:9: AssertionError"}, "start": 1761279511154, "stop": 1761279511154, "uuid": "53f6689e-8a88-49c0-b890-2c3a517fb439", "historyId": "c60a748c85eb19e0f24300418c1e1cbb", "testCaseId": "c60a748c85eb19e0f24300418c1e1cbb", "fullName": "tests.test_skip_xfail#test_known_bug", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_skip_xfail"}, {"name": "host", "value": "ASCENDI-QI7C07E"}, {"name": "thread", "value": "304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_skip_xfail"}], "titlePath": ["tests", "test_skip_xfail.py"]}